### PR TITLE
docs (core): add insights metrics widget to readme

### PR DESCRIPTION
**Pull request by jerdog**
In addition to the existing badges which show the status of builds, etc., this new widget that shows the Insights metrics for your repo that shows how the default_workflow is performing. More info can be found at: https://discuss.circleci.com/t/try-insights-snapshot-in-your-gh-repo/43169/

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [] Tests for the changes have been added (for bug fixes / features)
- [] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Regular build-related badges are displayed, but nothing to show how builds are performing over time

Issue Number: N/A

## What is the new behavior?
Display build stats on README

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by josephperrott — DISMISSED** [review]
LGTM

**Comment by jessicajaniuk** [comment]
@jerdog This is super close and just needs some lint fixes. Can you address and push? We should be able to land it right after that.

**Comment by jerdog** [comment]
@jessicajaniuk I checked and these errors have nothing to do with the file I submitted (README.md) and are not something I can fix.

**Comment by jessicajaniuk** [comment]
@jerdog It's likely the last commit in your PR that merges master in with your branch. You should drop that commit and rebase to master instead.

**Comment by jerdog** [comment]
Let me know @jessicajaniuk if I'm missing something, but all the checks pass now

**Comment by dylhunn** [comment]
Hi @jerdog. I'm sorry we stopped replying on this. Would you still like to merge it? If so, would you rebase it and then @ me for merge? Thanks!

**Review by dylhunn — APPROVED** [review]

**Comment by jerdog** [comment]
@dylhunn I just pulled the latest angular main branch into mine - I think we're good now?

**Comment by dylhunn** [comment]
@jerdog Awesome! Can you also squash your last three commits to clean up the commit log? You should be able to do something like this:

Remove the last 3 commit messages, while keeping the changes to the files:
```
git reset --soft HEAD~3
```

Create a new commit with all the changes:
```
git commit -a
```

**Comment by jerdog** [comment]
@dylhunn should be it lol

**Comment by dylhunn** [comment]
@jerdog Oh no! It looks like this PR now touches thousands of files (!!)

You're having weird issues because this PR was created before we renamed `master` to `main`. It might be easier to just make a new PR and commit with the same change? Since it has already been approved, just @ me and I can merge it immediately

**Comment by dylhunn** [comment]
Oh wait maybe it's fixed now?

File: README.md [hunk 1]
```diff
@@ -35,6 +35,12 @@
   </a>
 </p>
 
+<p align="center">
+  <a href="https://app.circleci.com/insights/github/angular/angular/workflows/default_workflow?branch=master">
```
Comment (dylhunn) [inline README.md:39]: This should be `main` now, right?

Comment (jerdog) [reply README.md:39]: Yeah - I'll redo this on `main` - looks like that's caused some issues and will just make this all easier :P 

I'll "@" you shortly

Comment (jerdog) [reply README.md]: Actually - @dylhunn I'm trying to just adjust this to `main` and I think we'll be good. I didn't even notice the (thankful) change to `main`

Comment (dylhunn) [reply README.md]: Awesome. I'll go ahead and merge it.

**Comment by dylhunn** [comment]
@jerdog Hm, the branch rename is still really confusing the merge tool. Mind pasting the change into a fresh commit and PR? Sorry for the trouble, something about pre-rename PRs just causes myriad issues

**Pull request closed without merging**